web/src/components/profile/ProfileErrorAlert.tsx

```diff
@@ -0,0 +1,61 @@
+import { Alert } from "@/components/njwds-extended/Alert";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { getProfileConfig } from "@/lib/domain-logic/getProfileConfig";
+import { ProfileContentField } from "@/lib/types/types";
+import { templateEval } from "@/lib/utils/helpers";
+import { ReactElement } from "react";
+
+interface Props {
+  fieldErrors: string[];
+}
+
+export const ProfileErrorAlert = (props: Props): ReactElement => {
+  const { Config } = useConfig();
+
+  const displayProfileErrorAlert = (): boolean => props.fieldErrors.length > 0;
+
+  const getHeader = (field: string): string => {
+    return getProfileConfig({
+      config: Config,
+      fieldName: field as ProfileContentField,
+    }).header;
+  };
+
+  const errorFieldsIds = (): string[] => {
+    const errorFieldsIds: string[] = [];
+    for (const error of props.fieldErrors) {
+      if (error === "willSellPetCareItems" && !errorFieldsIds.includes("petCareHousing")) {
+        errorFieldsIds.push("petCareHousing");
```

Comment:
I'm assuming `perCareHousing` is the only field where the error and the field name don't match?

Reply:
Why don't they match? This seems like a weird thing to have a special exception for, epsecially in a generic component like this. 

And if we really do need this, it should be tested

Reply:
I see now it was tested in the profile-shared but if I'd argue it belongs in the unit test for this component

Reply:
so basically only `petCareHousing` has the header field in the config because both the questions `willSellPetCareItems` and `petCareHousing` go under the same Petcare Services section. I will make it a unit test though.